kubetest/conformance/conformance.go

```diff
@@ -94,6 +96,29 @@ func NewDeployer(kubecfg string) (*Deployer, error) {
 	}, nil
 }
 
+func (_ *Deployer) GetKubectlVersion() error {
+	var (
+		interrupt = time.NewTimer(time.Duration(0)) // interrupt testing at this time.
+		terminate = time.NewTimer(time.Duration(0)) // terminate testing at this time.
+		verbose   = false
+		timeout   = time.Duration(0)
+		control   = process.NewControl(timeout, interrupt, terminate, verbose)
+		retries   = 5
+	)
+	for {
+		_, err := control.Output(exec.Command("./cluster/kubectl.sh", "--match-server-version=false", "version"))
+		if err == nil {
+			return nil
+		}
+		retries--
+		if retries == 0 {
+			return err
+		}
+		log.Print("Failed to reach api. Sleeping for 10 seconds before retrying...")
+		time.Sleep(10 * time.Second)
+	}
```

Comment:
I would say conformance deployer should do noop?

Reply:
also @BenTheElder we can probably get rid of the old dind conformance deployer?

Reply:
@krzyzacy @BenTheElder Just removed and made it no-op in `kubetest/conformance`. Thanks!

Reply:
yeah I don't think anyone is using the conformance deployer, I'll figure out something else for kind very soon...